BrightID-rn/src/store/index.js

```diff
@@ -1,10 +1,50 @@
 //store/index.js
 import thunkMiddleware from "redux-thunk";
-import { createStore, applyMiddleware } from "redux";
+import {createStore, applyMiddleware} from "redux";
 import reducer from "../reducer";
+import {setPPKeys} from "../actions/index";
+import nacl from "tweetnacl"
+import axios from "axios"
 
 const store = createStore(reducer, applyMiddleware(thunkMiddleware));
 
+export const setupPPKeys = ppKeys => async dispatch => {
+
+    try {
+
+        if (ppKeys.hasOwnProperty("key1") && ppKeys.hasOwnProperty("key2")) {
```

Comment:
Only need one key pair.

Reply:
yeah, I was going to speak to you about that. I noticed on the brightside-node publicKey1 and 2. We should chat about how the exchange will work.

Reply:
The exchange of public keys and signed messages is done when a connection is made.  See #75 .  This task is about creating one set of keys for one user to be used for making connections later.